ARM-wvd-templates/AddVirtualMachinesToHostPool/AddVirtualMachinesTemplate.json

```diff
@@ -1,502 +1,576 @@
-            "type": "string",
-            "metadata": {
-                "description": "The resource group containing the existing virtual network."
-            }
-        },
-        "createNetworkSecurityGroup": {
-            "type": "bool",
-            "metadata": {
-                "description": "Whether to create a new network security group or use an existing one"
-            },
-            "defaultValue": false
-        },
-        "networkSecurityGroupId": {
-            "type": "string",
-            "metadata": {
-                "description": "The resource id of an existing network security group"
-            },
-            "defaultValue": ""
-        },
-        "networkSecurityGroupRules": {
-            "type": "array",
-            "metadata": {
-                "description": "The rules to be given to the new network security group"
-            },
-            "defaultValue": []
-        },
-        "availabilitySetTags": {
-            "type": "object",
-            "metadata": {
-                "description": "The tags to be assigned to the availability set"
-            },
-            "defaultValue": {}
-        },
-        "networkInterfaceTags": {
-            "type": "object",
-            "metadata": {
-                "description": "The tags to be assigned to the network interfaces"
-            },
-            "defaultValue": {}
-        },
-        "networkSecurityGroupTags": {
-            "type": "object",
-            "metadata": {
-                "description": "The tags to be assigned to the network security groups"
-            },
-            "defaultValue": {}
-        },
-        "virtualMachineTags": {
-            "type": "object",
-            "metadata": {
-                "description": "The tags to be assigned to the virtual machines"
-            },
-            "defaultValue": {}
-        },
-        "imageTags": {
-            "type": "object",
-            "metadata": {
-                "description": "The tags to be assigned to the images"
-            },
-            "defaultValue": {}
-        },
-        "deploymentId": {
-            "type": "string",
-            "metadata": {
-                "description": "GUID for the deployment"
-            },
-            "defaultValue": ""
-        },
-        "apiVersion": {
-            "type": "string",
-            "metadata": {
-                "description": "WVD api version"
-            },
-            "defaultValue": "2019-12-10-preview"
-        },
-        "ouPath": {
-            "type": "string",
-            "metadata": {
-                "description": "OUPath for the domain join"
-            },
-            "defaultValue": ""
-        },
-        "domain": {
-            "type": "string",
-            "metadata": {
-                "description": "Domain to join"
-            },
-            "defaultValue": ""
-        }
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",

```

Comment:
FYI some editors other than VS Code may not support this per the[ $schema docs here](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-syntax#template-format)

Reply:
would you please point out which section in the above link tells other that VS Code may not support?

Reply:
I think this change is fine, that doc I linked to says it will work with Visual studio code but that "Other editors (including Visual Studio) may not be able to process this schema."

Reply:
Thanks.